cpp/src/community/detail/common_methods.cuh

```diff
@@ -100,6 +100,27 @@ struct reduce_op_t {
   }
 };
 
+// a workaround for cudaErrorInvalidDeviceFunction error when device lambda is used
```

Comment:
Add `FIXME:` to better track this. We may not need this anymore once we update to CCCL 2.x (`cuda::proclaim_return_type<size_t>`)

Reply:
Added.